nodes/node_axle.lua

```diff
@@ -1,6 +1,7 @@
 -- internationalization boilerplate
-local MP = minetest.get_modpath(minetest.get_current_modname())
-local S = dofile(MP.."/intllib.lua")
+local S = digtron.S
+-- local MP = minetest.get_modpath(minetest.get_current_modname())
+-- local S = dofile(MP.."/intllib.lua")
```

Comment:
Feel free to remove those. They're no longer needed as Minetest's translation works pretty well so far and might even get .po (gettext) support in the future.